core/jazz_logs/index.js

```diff
@@ -59,7 +59,7 @@ module.exports.handler = (event, context, cb) => {
         return cb(JSON.stringify(errorHandler.throwInputValidationError("missing required input parameter category.")));
       }
 
-      if (!event.body.type || !_.includes(config.VALID_LOGTYPES, event.body.type.toLowerCase())) {
```

Comment:
I would prefer keeping this as-is. This would handle case insensitive searches (which is nice!).

Reply:
Right. In which case there is no need to change the case of the VALID_LOGTYPES.